### ISSUE TITLE
Training perioid

### ISSUE THREAD

**Issue by jyeatman** [SwipesForScience/SwipesForScience#88]
Add the possibility to have a training period where feedback is give on correct/incorrect trials and we train people until the get to XX% correct on really easy trials. In my case, I would like to have an animated character tell participants what they did wrong/right during the training period. This might be a good general feature. For example, a scientist explaining some aspect of the image that the participant missed in braindr or fiibr

**Comment by akeshavan** [comment]
Yes this is definitely a general good feature! Somewhere in this funciton is where i'd start: https://github.com/SwipesForScience/SwipesForScience/blob/master/src/components/Play.vue#L378-L419

its taking a look at the `userInfo` variable where you can store whatever you need to keep track of accuracy. The logic here can be edited to accomdate this
